<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleWindow</class>
 <widget class="QMainWindow" name="ConsoleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>384</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>192</red>
            <green>192</green>
            <blue>192</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>32</red>
            <green>32</green>
            <blue>32</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>192</red>
            <green>192</green>
            <blue>192</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>32</red>
            <green>32</green>
            <blue>32</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_save"/>
   <addaction name="action_clear"/>
   <addaction name="action_pause"/>
   <addaction name="action_wrap"/>
   <addaction name="separator"/>
   <addaction name="action_terminal"/>
  </widget>
  <action name="action_save">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/new/prefix1/Images/save.png</normaloff>:/new/prefix1/Images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/new/prefix1/Images/clear.png</normaloff>:/new/prefix1/Images/clear.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="action_wrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/new/prefix1/Images/wordwrap.png</normaloff>:/new/prefix1/Images/wordwrap.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Word Wrap</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/new/prefix1/Images/pause.png</normaloff>:/new/prefix1/Images/pause.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
  </action>
  <action name="action_terminal">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/new/prefix1/Images/application_xp_terminal.png</normaloff>:/new/prefix1/Images/application_xp_terminal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Terminal</string>
   </property>
   <property name="toolTip">
    <string>Start Terminal Program</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
